modules/bootconf.py

```diff
@@ -14,7 +14,7 @@
 
 def configureBootloader(old, new, conffile):
     ''' Configure bootloader to use the updated rootfs '''
-    if runningDevice(conffile) == 'raspberry-pi' or runningDevice(conffile) == 'raspberry-pi2':
+    if runningDevice(conffile) == 'raspberry-pi' or runningDevice(conffile) == 'raspberry-pi2' or runningDevice(conffile) == 'raspberrypi3':
```

Comment:
Can this be `if runningDevice(conffile) in [ 'raspberry-pi', 'raspberry-pi2', 'raspberrypi3' ]`?